src/main/scala/nl/knaw/dans/api/sword2/DepositProperties.scala

```diff
@@ -65,8 +65,8 @@ object DepositProperties {
   private def readProperties(f: File) = {
     val ps = new PropertiesConfiguration()
     ps.setDelimiterParsingDisabled(true)
-    ps.load(f)
-
+    if(f.exists) ps.load(f)
```

Comment:
Quick question: are you sure we still want an empty `PropertiesConfiguration` when the file doesn't exist? Haven't looked at the context, just wondering!